sofiadinizms/barbie-world
# feat(feature/notifications): created notifications

## What I did

- Created the notifications 

## How to test

- Clone this branch
- Run the project
- Check if the notifications work fine, check the time they are scheduled to appear and the trigger (if needed, call me)
